### DISCUSSION TITLE
Title instead of prompt for ui_select

### DISCUSSION THREAD

**Discussion by xyven1** [ibhagwan/fzf-lua#1471] [Q&A] [answered]
Despite having selected `borderless_full` as the preset for fzf-lua, the `ui_select` window still uses the prompt style. I can see in the code for the `ui_select` that the `[--prompt]` value is set for `fzf_opts`, so maybe this isn't configurable ATM?
Any and all ideas welcome.

**Comment by ibhagwan** [comment]
> so maybe this isn't configurable ATM? Any and all ideas welcome.

Probably an oversight, I’ll take a look.

**Comment by ibhagwan** [comment, marked as answer]
You’re correct, it’s currently not possible, the prompt is dynamic and is set per call:
https://github.com/ibhagwan/fzf-lua/blob/ebb89e4e4065e31b029eee8f618e1ca660f41f35/lua/fzf-lua/providers/ui_select.lua#L98-L99

Will probably require an setting, I’ll think about the best way to add this.
